Add a note to the README about manifest placeholders when using flavors (#394)
auth0/react-native-auth0: README.md

The `applicationId` value will be auto-replaced on runtime with the package name or id of your application (e.g. `com.example.app`). You can change this value from the `build.gradle` file. You can also check it at the top of your `AndroidManifest.xml` file.

> > Note that if you're using [flavors](https://developer.android.com/studio/build/build-variants#product-flavors) with different application IDs, you'll need to set up the manifest placeholders for each flavor.

Reviewer: Having different "application IDs" doesn't impact the manifest placeholder contents, as those are used just for domain and scheme.

Furthermore, properties defined in the root level of the `android` block would affect all product flavors. If one requires a different domain, for example, it would need to override it explicitly by defining it again inside their flavor block.

I think the sentence can be improved a bit this way:

```suggestion
> Note that if your Android application is using [product flavors](https://developer.android.com/studio/build/build-variants#product-flavors), you might need to specify different manifest placeholders for each flavor.
```

Author: > Having different "application IDs" doesn't impact the manifest placeholder contents, as those are used just for domain and scheme.

If the app has different app ids defined in each flavor, that definitely impacts the manifest placeholders, as the app id is used as the scheme in the intent filter.
